.github/workflows/release_linux_aarch64.yml

```
@@ -0,0 +1,80 @@
+name: LinuxRelease_aarch64
+
+on:
+  schedule:
+    # Run weekly on Monday 00:00
+    - cron:  '00 00 * * MON'
+  push:
+    branches: [master, rel-*]
+  pull_request:
+    branches: [rel-*]
```

```suggestion
    branches: [rel-*, master]  # TODO remove master before merge
```
Could you please add master trigger for testing? Thank you.

---

Done.